examples/pubsub/main.tf

```diff
@@ -36,26 +36,19 @@ resource "google_pubsub_topic_iam_member" "sm_sa_publisher" {
   topic   = google_pubsub_topic.secret.name
 }
 
-module "secret-manager" {
-  source  = "GoogleCloudPlatform/secret-manager/google"
+module "secret" {
+  source  = "GoogleCloudPlatform/secret-manager/google//modules/secret"
```

Comment:
Let's change the simple example but keep other showcasing root module

Reply:
OK, done.